apps/vmq_plugin/src/vmq_plugin_mgr.erl

```diff
@@ -390,7 +398,7 @@ check_updated_plugins(Plugins, State) ->
             ok = init_plugins_cli(CheckedPlugins),
             ok = start_plugins(CheckedPlugins),
             ok = compile_hooks(CheckedPlugins),
-            {ok, handle_deferred_calls(State#state{ready=true, plugins=CheckedPlugins})};
```

Comment:
This was an actual bug in `vmq_plugin_mgr` which actually never got triggered as we have deferred all calls.